sdmetrics/reports/multi_table/quality_report.py

```diff
@@ -226,124 +138,24 @@ def get_details(self, property_name, table_name=None):
-                    if table_name is not None and table != table_name:
-                        continue
-
-                    for column_pair, score_breakdown in table_breakdown.items():
-                        tables.append(table)
-                        columns.append(column_pair)
-                        metrics.append(metric.__name__)
-                        scores.append(score_breakdown.get('score', np.nan))
-                        real_scores.append(score_breakdown.get('real', np.nan))
-                        synthetic_scores.append(score_breakdown.get('synthetic', np.nan))
-                        errors.append(score_breakdown.get('error', np.nan))
-
-            details = pd.DataFrame({
-                'Table': tables,
-                'Column 1': [col1 for col1, _ in columns],
-                'Column 2': [col2 for _, col2 in columns],
-                'Metric': metrics,
-                'Quality Score': scores,
-                'Real Correlation': real_scores,
-                'Synthetic Correlation': synthetic_scores,
-            }).sort_values(by=['Table'], ignore_index=True)
-
-        elif property_name == 'Cardinality' or 'Parent Child Relationships':
-            if property_name == 'Parent Child Relationships':
-                property_name = 'Cardinality'
-                msg = (
-                    "The 'Parent Child Relationships' property name is no longer recognized. "
-                    "Please update to 'Cardinality' instead.")
-                warnings.warn(msg, FutureWarning)
-
-            child_tables = []
-            for metric in self.METRICS[property_name]:
-                for table_pair, score_breakdown in self._metric_results[metric.__name__].items():
-                    if table_name is not None and table_name not in table_pair:
-                        continue
-
-                    tables.append(table_pair[0])
-                    child_tables.append(table_pair[1])
-                    metrics.append(metric.__name__)
-                    scores.append(score_breakdown.get('score', np.nan))
-                    errors.append(score_breakdown.get('error', np.nan))
-
-            details = pd.DataFrame({
-                'Child Table': child_tables,
-                'Parent Table': tables,
-                'Metric': metrics,
-                'Quality Score': scores,
-            })
-
-        if pd.Series(errors).notna().sum() > 0:
-            details['Error'] = errors
+        self._validate_inputs(property_name, table_name)
 
-        return details
+        property_instance = self._properties_instances[property_name]
+        if property_name != 'Cardinality':
+            if table_name:
+                return property_instance._properties[table_name]._details.copy()
 
-    def get_raw_result(self, metric_name):
-        """Return the raw result of the given metric name.
+            details = {}
+            for table_name, properties in property_instance._properties.items():
```

Comment:
minor: I think it should be table_name, property since you're dealing with a single property at this point